javalin/src/main/java/io/javalin/http/HttpCode.kt

```diff
@@ -63,13 +63,28 @@ enum class HttpCode(val status: Int, val message: String) {
     LOOP_DETECTED(508, "Loop Detected"),
     @Deprecated("Obsoleted by https://datatracker.ietf.org/doc/status-change-http-experiments-to-historic/")
     NOT_EXTENDED(510, "Not Extended"),
-    NETWORK_AUTHENTICATION_REQUIRED(511, "Network Authentication Required");
+    NETWORK_AUTHENTICATION_REQUIRED(511, "Network Authentication Required"),
+    CUSTOM_HTTP_CODE(599, "Non-standard HTTP code");
```

Comment:
Can accept any custom HTTP Codes, I don't really like the use of a custom enum with value 599, but it works fine when obtained through the `forStatus()` function.